### ISSUE TITLE
Taxon constraint:  GO:0120020 | cholesterol transfer activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#27276]
Please provide as much information as you can: 

* **GO term ID label:**

GO:0120020 | cholesterol transfer activity

* **Request to add a taxon constraint:**

* * **never in taxon: **

never in yeast (or fungi?)

* **Request to remove a taxon constraint:**
Please specify

* **Supporting evidence if available (e.g PMID):**

**Comment by ValWood** [comment]
Also 

GO:0015485 | cholesterol binding |

**Comment by ValWood** [comment]
cholesterol is absent in fungi,

also
Cholesterol is [biosynthesized](https://en.wikipedia.org/wiki/Biosynthesis) by all animal [cells](https://en.wikipedia.org/wiki/Cell_(biology)#Eukaryotic_cells) and is an essential structural component of [animal](https://en.wikipedia.org/wiki/Animal) [cell membranes](https://en.wikipedia.org/wiki/Cell_membrane). In [vertebrates](https://en.wikipedia.org/wiki/Vertebrate), [hepatic cells](https://en.wikipedia.org/wiki/Hepatocyte) typically produce the greatest amounts. In the brain astrocytes produce cholesterol and transport it to neurons.[[5]](https://en.wikipedia.org/wiki/Cholesterol#cite_note-5) It is absent among [prokaryotes](https://en.wikipedia.org/wiki/Prokaryote) ([bacteria](https://en.wikipedia.org/wiki/Bacteria) and [archaea](https://en.wikipedia.org/wiki/Archaea)), although there are some exceptions, such as [Mycoplasma](https://en.wikipedia.org/wiki/Mycoplasma), which require cholesterol for growth.[[6]](https://en.wikipedia.org/wiki/Cholesterol#cite_note-6) Cholesterol also serves as a [precursor](https://en.wikipedia.org/wiki/Precursor_(chemistry)) for the [biosynthesis](https://en.wikipedia.org/wiki/Biosynthesis) of [steroid hormones](https://en.wikipedia.org/wiki/Steroid_hormone), [bile acid](https://en.wikipedia.org/wiki/Bile_acid)[[7]](https://en.wikipedia.org/wiki/Cholesterol#cite_note-Hanukoglu_1992-7) and [vitamin D](https://en.wikipedia.org/wiki/Vitamin_D).

so I only feel comfortable adding "never in fungi" and "never in archaea"

**Comment by deustp01** [comment]
> cholesterol is absent in fungi,

But does this mean that fungi confronted with exogenous cholesterol do not normally transport it? (It might - this is a question out of ignorance.)

**Comment by ValWood** [comment]
gosh, not sure.
but this term

Definition | Parents
-- | --
Removes cholesterol from a membrane or a monolayer lipid particle, transports it through the aqueous phase while protected in a hydrophobic pocket, and brings it to an acceptor membrane or lipid particle.

which is intracellular and act  in yeast on ergosterol that is already in the membrane, I think we are probably safe to add this restriction. i.e.  if there is transport of cholesterol it is probably not by an ergosterol transfer protein in the ER membrane acting on endogenous sterol.

We will probably benefit more from restriction to prevent the predictions  errors  (most cholesterol terms are already restricted for many years) and refine if we come across a published example.  Maybe some necrotrophs do it? When I checked I could not find anything.

**Comment by ValWood** [comment]
@pgaudet  reopen if  you think I should revert this

**Comment by pgaudet** [comment]
This is trasnfer within the membrane. Yeast probably does that for ergosterol but not cholesterol ?

**Comment by ValWood** [comment]
It's intermembrane in this instance, PM to ER, but I agree that as far as we know these are selective for ergosterol (otherwise we would have mosaic membranes....). So for now we can keep like this.

**Comment by pgaudet** [comment]
I dont see any PR associated with this; did you add a TC?

**Comment by ValWood** [comment]
Yes, I'm sure it was in one of them.... I'm out this morning will double check  when I get back

**Comment by ValWood** [comment]
I see it in here:
https://github.com/geneontology/go-ontology/edit/master/src/taxon_constraints/never_in_taxon.tsv

**Comment by pgaudet** [comment]
Perfect, thanks 

Please if yo u can link pull requests to the appropriate ticket it's easier to follow up on what was done/not done.
